src/data/dtsi/queries/queryDTSIAllPeople.ts

```diff
@@ -32,14 +20,21 @@ const query = /* GraphQL */ `
 /*
 Because this request returns so many results, we should ensure we're only triggering this logic in cached endpoints/routes
 */
-export const queryDTSIAllPeople = async ({ limit }: { limit: number } = { limit: 1500 }) => {
+export const queryDTSIAllPeople = async ({
+  limit = 1500,
+  countryCode = SupportedCountryCodes.US,
```

Comment:
it's a neat, but keeps consistency with other implementations we already have

```suggestion
  countryCode = DEFAULT_SUPPORTED_COUNTRY_CODE,
```